ci-operator/step-registry/ipi/conf/azure/resourcegroup/ipi-conf-azure-resourcegroup-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -x
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-existingresourcegroup.yaml.patch"
+
+azure_region=$(/tmp/yq r "${CONFIG}" 'platform.azure.region')
+cluster_name=$(/tmp/yq r "${CONFIG}" 'metadata.name')
+existing_rg=${cluster_name}-exrg
+
+# az should already be there
```

Comment:
it's not.... at the moment: 
```
+ az version
/bin/bash: line 20: az: command not found
```
looks like you're using `centos` image, you should probably be using the upi image